Repositories/FavouriteColourRepository.cs

```diff
@@ -0,0 +1,36 @@
+using System.Data;
+using ColoursTest.Models;
+using Dapper;
+
+namespace ColoursTest.Repositories
+{
+    class FavouriteColourRepository : IFavouriteColourRepository
```

Comment:
Since favorite colours belong to a person I would probably deal with adding or removing them when updating the person rather than through a separate repository, again if you think about what would happen if you switched to MongoDB, for example, you may just have a single document rather than multiple relational tables which may potentially render this repository useless.

Reply:
Okay, in this instance I could move it to the People repository but how would I do a transaction across multiple repositories? Or is this not possible?

Reply:
this is possible and to achieve this you would probably use the Unit Of Work pattern and with this, you may need to pass in the connection or something similar in that case.